docs/pages/Models.md

```diff
@@ -399,7 +403,7 @@ random draw from a Gaussian, or combinations of the two.
 
 <span id="models_in_progress" class="anchor"></span>  
 
-## Models that are supported by DART Classic and could be ported to the Manhattan release if needed.
+## Models that are supported by DART Classic (*Lanai*) and could be ported to the Manhattan release if needed.
```

Comment:
classic and lanai are different branches - maybe add "or" in front of lanai?

(edit: i know hardly anyone ever used classic, but they are different)